uber/templates/registration/form.html

```diff
@@ -146,6 +146,27 @@
         </div>
     </div>
 
+    <div class="form-group">
+        <label for="admin_status" class="col-sm-2 control-label">Admin</label>
+        <div class="col-sm-6">
+            {% if attendee.admin_account %}
+                <p name="admin_status">True</p>
+            {% else %}
+                <p name="admin_status">False</p>
+            {% endif %}
+        </div>
+    </div>
+
+    {% if attendee.admin_account %}
```

Comment:
This can probably just replace the block above, OR you can combine it into something where it shows the access_labels if they have an admin account or "No admin account" otherwise.